launch.sh

```diff
@@ -10,11 +10,37 @@ until pg_isready; do
 done
```

Comment:
This `pg_isready` only contacts localhost, does not even check for the apollo database which takes much longer to setup. This is something that would happen no matter if connecting to local / remote. Have you compiled and run this image? Was it functional without a separate pg_isready? Maybe you got lucky on startup times.